docs/typescript/quickstart/node-fetch.md

```diff
@@ -0,0 +1,120 @@
+---
+id: node-fetch
+title: Sample Application with the typescript SDK
+sidebar_label: Node-fetch sample
+tags:
+  - node-fetch
+  - typescript
+  - sdk
+  - tutorial
+---
+
+#  Sample Node-fetch App 
+
+A sample node-fetch app containing GET and POST request integrated with keploy's [typescript-sdk](https://github.com/keploy/typescript-sdk)
+
+## Installation
+
```

Comment:
@mukul314 can you include the installation part here instead which you've added [here](https://github.com/keploy/docs/pull/162/files#diff-68f73088dc8486a51efc8eda10a3d5165a5173eaa8569e38c5656d0d17d5cd0cR25) like shown below.
![Screenshot from 2023-04-04 20-36-53](https://user-images.githubusercontent.com/74777863/229837937-ec5983d2-5d5f-420c-85bf-ef3a7c5ae475.png)